sites/all/themes/orwell/sass/component/_header.scss

```diff
@@ -30,6 +30,11 @@
   display: none;
 }
 
+// But show it if login just failed.
+.login-failed .js-login-form {
```

Comment:
You shouldnt use js- selectors in CSS :)

Reply:
But we have a

```
 .js .js-login-form {
    display: none;
  }
```

Just above. I believe to eliminate load flashing (Drupal behaviours are triggered at document load). In retrospect, the class should probably have another name.

Reply:
Ok, i think the best would be for you to fix it, but i'll approve it even if you dont 👍